### ISSUE TITLE
[Feature] Show text below progress widget 

### ISSUE THREAD

**Issue by ReniDelonzek** [arsamme/flutter-progress-dialog#2]
Sometimes it is interesting to present a text to the user informing the action being taken. The suggestion is to add an optional parameter in which we can pass a widget to display just below the progress

**Comment by arsamme** [comment]
@ReniDelonzek 
You can pass your own widget in `loadingWidget `parameter.

` ArsProgressDialog customProgressDialog = ArsProgressDialog(
	 context,
	 blur: 2,
	 backgroundColor: Color(0x33000000),
	 loadingWidget: Container(
		 width: 150,
		 height: 150,
		 color: Colors.red,
		 child: CircularProgressIndicator(),    
    ));`

**Comment by ReniDelonzek** [comment]
Yes, it would be interesting to have direct support and this feature without having to overwrite the loading. But feel free to close this request

**Comment by arsamme** [comment]
I'll add in next release
